lib/main.ts

```diff
@@ -45,6 +45,7 @@ export interface TranspilerOptions {
 
 export const COMPILER_OPTIONS: ts.CompilerOptions = {
   allowNonTsExtensions: true,
+  experimentalDecorators: true,
```

Comment:
as discussed, keeping this as it is, since don't have a single user who would want experimentalDecorators: false